tests/test_handler_DataHandler.py

```diff
@@ -103,11 +103,11 @@ def test_parse_format_line():
 
     fmt = 'FORMAT datatype=RNA missing=? symbols="ACGU" labels interleave;'
     f = d.parse_format_line(fmt)
-    assert f['datatype'] == 'rna', \
+    assert f['datatype'] == 'RNA', \
         "Expected 'rna', but got '%s'" % f['datatype']
     assert f['missing'] == '?', \
         "Expected '?', but got '%s'" % f['missing']
-    assert f['symbols'] == 'acgu', \
+    assert f['symbols'] == 'ACGU', \
```

Comment:
So this change affects more tests. But for the case of `symbols`, I'd argue that lowercasing would be a bug (unless we also want to lowercase the sites, and read and apply the `RespectCase` format ...)

Reply:
agree, it's a bug.